src/main/java/roomescape/service/ReservationService.java

```diff
@@ -0,0 +1,37 @@
+package roomescape.service;
+
+import lombok.RequiredArgsConstructor;
+import org.springframework.stereotype.Service;
+import org.springframework.util.StringUtils;
+import roomescape.domain.Reservation;
+import roomescape.exception.ParamException;
+import roomescape.exception.ReservationException;
+import roomescape.repository.JdbcTemplateReservationRepository;
+
+import java.util.List;
+
+@Service
+@RequiredArgsConstructor
+public class ReservationService {
+    private final JdbcTemplateReservationRepository jdbcTemplateReservationRepository;
+
+    public List<Reservation> getAllReservations() {
+        return jdbcTemplateReservationRepository.findAll();
+    }
+    public Reservation addReservation(Reservation reservation) {
+
+
+        if (StringUtils.isEmpty(reservation.name()) || StringUtils.isEmpty(reservation.date()) || StringUtils.isEmpty(reservation.time())) {
+            throw new ParamException(ParamException.Type.MISSING_PARAMETER);
+        }
```

Comment:
위와 비슷한 이야기인데, 문장이 길어질 때는 아래처럼 규칙적으로 직접 개행해주는 것도 좋다고 생각합니다. 
```suggestion
        if (StringUtils.isEmpty(reservation.name()) || 
            StringUtils.isEmpty(reservation.date()) || 
            StringUtils.isEmpty(reservation.time())) {
            throw new ParamException(ParamException.Type.MISSING_PARAMETER);
        }
```